source/blog/2016-07-14-new-centos-atomic-host-releases-available-for-download.html.md

```diff
@@ -23,9 +23,13 @@ CentOS Atomic Host includes these core component versions:
 CentOS Atomic Host is available as a VirtualBox or libvirt-formatted Vagrant box, or as an installable ISO, qcow2 or Amazon Machine image. Check out the [CentOS wiki](https://wiki.centos.org/SpecialInterestGroup/Atomic/Download) for download links and installation instructions, or read on to learn more about what's new in this release.
 
 ## OCI Advances: runC and OCI Hooks
-CentOS Atomic Host now ships with [runC](http://runc.io/), a lightweight client wrapper around libcontainer for spawning and running containers according to the [Open Container Initiative](https://www.opencontainers.org/) specification. To learn about what you can do with runC and OCI, check out [this post](http://www.projectatomic.io/blog/2016/04/running_OCI/) from [Mrunal Patel](https://twitter.com/mrunalp), and this [Dockercon 2016 talk](https://www.youtube.com/watch?v=ZAhzoz2zJj8) from [Phil Estes](https://twitter.com/estesp).
+CentOS Atomic Host now ships with [runC](http://runc.io/), a lightweight client wrapper around libcontainer for spawning and running containers according to the [Open Container Initiative](https://www.opencontainers.org/) specification. In this release, the host's atomic run tool makes use of runC to manage system containers, also known as [super privileged containers](http://developerblog.redhat.com/2014/11/06/introducing-a-super-privileged-container-concept/). While it's possible to run system containers with docker, there are certain system components that present chicken-and-egg scenarios, such as flannel, on which docker itself depends.
```

Comment:
This is how Giuseppe IDs them in docker hub: https://hub.docker.com/r/gscrivano/spc-flannel/. I can take that bit out, though.